sdk/devcenter/azure-developer-devcenter/sdk_packaging.toml

```diff
@@ -0,0 +1,10 @@
+[packaging]
+package_name = "azure-developer-devcenter"
+package_nspkg = "azure-developer-nspkg"
+package_pprint_name = "MyService Management"
```

Comment:
This is correct? Should be "Azure Developer DevCenter Service"?

Reply:
@changlong-liu not our changes - this is another change that AutorestCI bot is adding to the PR. Our changes are extremely minimal and can be found in this commit: https://github.com/Azure/azure-sdk-for-python/pull/27308/commits/f41ec2172de7009991dadaaa48f7e4cbb6f036e0

Please let us know how we can disable this bot from running, or how we can get it to run properly.

Reply:
Fixed after disabling push from AutorestCI.